docs/background/plot_01_1D_basis_function.py

```diff
@@ -52,8 +82,8 @@
 #
 # Let's see how this two modalities operate.
 
-eval_mode = nmo.basis.BSplineBasis(n_basis_funcs=n_basis, mode="eval")
-conv_mode = nmo.basis.BSplineBasis(n_basis_funcs=n_basis, mode="conv", window_size=100)
+eval_mode = nmo.basis.MSplineBasis(n_basis_funcs=n_basis, mode="eval")
```

Comment:
to switch it up, and not show a single case. before it was MSpline then BSpline for the same reason. Now I prefer BSpline for the first example, because the MSpline scales differently with the range.

Eventually, we will need a dedicated doc page for basis in which we show each one.